tasks/cephfs/test_volume_client.py

```diff
@@ -23,11 +23,11 @@ def _volume_client_python(self, client, script):
 log = logging.getLogger("ceph_volume_client")
 log.addHandler(logging.StreamHandler())
 log.setLevel(logging.DEBUG)
-vc = CephFSVolumeClient("manila", "{conf_path}", "ceph")
+vc = CephFSVolumeClient("manila", "{conf_path}", "ceph", {vol_prefix}, {ns_prefix})
```

Comment:
The fourth and fifth arguments would have to be "{vol_prefix}" if "{vol_prefix}" else None, "{ns_prefix}" if "{ns_prefix}" else None, respectively. If you can come up with something more elegant to fix this issue it'd be great. Or maybe you can pass arguments with quotes escaped to this function, e.g  in line 115, vol_prefix="\"/myprefix\"" .